File: TemplateVB/Addin/AddinRibbonComponent.vb
Imports Microsoft.Office.Tools.Ribbon

Public Class AddinRibbonComponent

    Private Sub Command1_Click(sender As System.Object, e As Microsoft.Office.Tools.Ribbon.RibbonControlEventArgs) Handles Command1.Click
        Globals.ThisAddIn.Command1()
    End Sub
    $if$ ($taskpane$ == true)
    Private Sub TogglePanel_Click(sender As System.Object, e As Microsoft.Office.Tools.Ribbon.RibbonControlEventArgs) Handles TogglePanel.Click
        Globals.ThisAddIn.TogglePanel()
    End Sub
$endif$
End Class

File: TemplateCS/Addin/AddinRibbonComponent.cs
using System;
using System.Collections.Generic;
using Microsoft.Office.Tools.Ribbon;

namespace $csprojectname$
{
    public partial class AddinRibbonComponent
    {
        private void buttonCommand1_Click(object sender, RibbonControlEventArgs e)
        {
            Globals.ThisAddIn.Command1();
        }
        $if$ ($taskpane$ == true)
        private void buttonToggle_Click(object sender, RibbonControlEventArgs e)
        {
            Globals.ThisAddIn.TogglePanel();
        }
    $endif$}
}
